pkg/logs/logs.go

```diff
@@ -0,0 +1,35 @@
+package logs
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/common"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+func StoreTestLogs(testNamespace, jobName string, componentPipelineRun *v1beta1.PipelineRun, cs *common.SuiteController, t *tekton.SuiteController) error {
+	wd, _ := os.Getwd()
+	artifactDir := GetEnv("ARTIFACT_DIR", fmt.Sprintf("%s/tmp", wd))
+	testLogsDir := fmt.Sprintf("%s/%s", artifactDir, testNamespace)
+
+	if err := os.MkdirAll(testLogsDir, os.ModePerm); err != nil {
+		return err
+	}
+
+	if err := cs.StorePodLogs(testNamespace, jobName, testLogsDir); err != nil {
+		GinkgoWriter.Printf("Failed to store pod logs: %v", err)
```

Comment:
I think we should return an error here. WDYT?

```suggestion
                GinkgoWriter.Printf("Failed to store pod logs: %v", err)
                return err
```

Reply:
If we return an error at this point, we won't be able to store pipelineRun logs. Would it be sufficient to join errors and return them at the end of the StoreTestLogs function?

Reply:
Yes, makes sense!